TryAtSoftware.Extensions.Collections/Bitmask.cs

```diff
@@ -0,0 +1,100 @@
+﻿namespace TryAtSoftware.Extensions.Collections;
+
+using System;
+using System.Collections.Generic;
+
+public class Bitmask
+{
+    public const int BitsPerSegment = 64;
+    private readonly List<long> _segments;
```

Comment:
C# has a type called `UInt128`. Maybe it should be used to reduce the number of segments.